build-system/tasks/bundle-size/README.md

```diff
@@ -0,0 +1,24 @@
+# Bundle Size checks
+
+The `APPROVERS.json` file is a listing of compiled AMP runtime and extensions
+files. Each rule has two parts:
+
+- `approvers`: list of GitHub project teams, members of which are allowed to
+  approve pull requests that increase the file's bundle size (Brotli-compressed
+  size)
+- `threshold`: number of KBs that the file can increase before requiring an
+  approval from the above team members.
```

Comment:
Nit: I'd move the Brotli-compressed bit out of approvers and into threshold, ie.  `number of kilobytes by which the file's Brotli-compressed size may increase without require approval` or something of that nature

Reply:
Done